src/crimson/os/seastore/cache.cc

```diff
@@ -859,6 +859,7 @@ CachedExtentRef Cache::duplicate_for_write(
     return i;
 
   auto ret = i->duplicate_for_write();
+  ret->last_modified = ceph::real_clock::now();
```

Comment:
seastar/core/lowres_clock.hh might be a better choice since it introduces less overhead and the time doesn't need to be very accurate.

Reply:
It seems that lowres clock 's granularity is 10ms, I'm worried whether it's precise enough. I think maybe we keep ceph::real_clock for now, and change it if it proves to be a perf bottleneck in the future, what do you think?